### PR TITLE
feat: document password migration hook

### PR DESCRIPTION
**Pull request by hperl**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/kratos/manage-identities/25_import-user-accounts-identities.mdx [hunk 1]
```diff
@@ -453,6 +453,86 @@ $hmac-sha512$OTFmODY0ZTI1NmU0ZjVhYjhiMDViZGFmNGVmNGZmMGVlNTY4ODYwNWJhYTk4MTk2OTg
...
+
+Add the web hook URL to the configuration. See the
+[web hook configuration](https://www.ory.sh/docs/guides/integrate-with-ory-cloud-through-webhooks#request-authentication) for
+authentication options.
+
+```yaml
+selfservice:
+  flows:
+    login:
+      password_migration:
```
[the remark this answers is not in the document]
Comment (jonas-jonas) [reply docs/kratos/manage-identities/25_import-user-accounts-identities.mdx:498]: Agreed, is this still possible to change in Kratos? @hperl

File: docs/kratos/manage-identities/25_import-user-accounts-identities.mdx [hunk 2]
```diff
@@ -453,6 +453,86 @@ $hmac-sha512$OTFmODY0ZTI1NmU0ZjVhYjhiMDViZGFmNGVmNGZmMGVlNTY4ODYwNWJhYTk4MTk2OTg
...
+
+If you want to import users, but do not have access to the hashed password, or the hashed password is in a format that Ory
+Identities does not support, you can use a web hook to migrate the password.
+
+The web hook will be called when the user logs in, and will receive the user's identifier and password. If the web hook returns
+successfully, the user will be logged in and the hashed password will be stored.
+
+The following steps are necessary to set up password migration using a web hook:
+
+#### Step 1: Import an identity with an empty password hash and `use_migration_hook` set to `true`
```
Comment (jonas-jonas) [inline docs/kratos/manage-identities/25_import-user-accounts-identities.mdx:466]: For "stepped" guides, we use lists in other places. IMO, this should be consistent across all docs, so I'd prefer a list here, similar to here: https://www.ory.sh/docs/hydra/guides/custom-ui-oauth2#run-the-sample-application
(source: https://github.com/ory/docs/blob/master/docs/hydra/guides/custom-ui-oauth2.mdx#L105-L133)

**Review by jonas-jonas — APPROVED** [review]

**Review by jonas-jonas — CHANGES_REQUESTED** [review]
Wrongly approved, please see above.

File: docs/kratos/manage-identities/25_import-user-accounts-identities.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hperl) [reply docs/kratos/manage-identities/25_import-user-accounts-identities.mdx]: ok fixed

File: docs/kratos/manage-identities/25_import-user-accounts-identities.mdx (hunk 2 above)
Comment (hperl) [reply docs/kratos/manage-identities/25_import-user-accounts-identities.mdx]: fixed

**Review by aeneasr — APPROVED** [review]

**Pull request merged**